Source/JavaScriptCore/b3/B3OptimizeAssociativeExpressionTrees.cpp

```diff
@@ -191,7 +191,7 @@ bool OptimizeAssociativeExpressionTrees::optimizeRootedTree(Value* root, Inserti
         return false;
     }
 
-    std::sort(leaves.begin(), leaves.end(), [](Value* x, Value* y) {
+    std::ranges::sort(leaves, [](Value* x, Value* y) {
```

Comment:
Can update this to use auto instead.

```suggestion
    std::ranges::sort(leaves, [](auto* x, auto* y) {
```